vassal-app/src/main/java/VASSAL/configure/BeanShellExpressionConfigurer.java

```diff
@@ -323,6 +323,18 @@ protected void updateParentBuilder() {
     }
   }
 
+  protected Font getEditorFont(int style, int size) {
+    try {
+      final String fileName = "/fonts/JetBrainsMono.ttf";
+      final URL url = getClass().getResource(fileName);
+      final Font ms = Font.createFont(Font.TRUETYPE_FONT, url.openStream());
+      return ms.deriveFont(style, size);
+    }
+    catch (Exception ignored) {
+    }
+    return new Font("monospaced", style, size);
+  }
+
```

Comment:
It looks like you got some font changes in here accidentally.

Reply:
Not so accidentally :) This is the minimal font additional code to provide a decent font in the Expression editor. Completely compatible with what I want to do later. Plus it will annoy the shit out of P until 3.8.

Reply:
This will prevent the system monospace font from being used if the hard-coded font is present in one of the JARs, right? Is that really what we want?

Reply:
For the one component where getEditorFont() is called, yes. that's the point. I'll make it a configurable selection in 3.8. It won't stop the system monospace font being used anywhere else.

Reply:
I think I don't understand what this is for, in that case. Why do we want to use something other than the system default _by default_?